README.md

```diff
@@ -33,6 +33,8 @@ Table of Contents
     * [resty.kong.log.set\_log\_level](#restykonglogset_log_level)
     * [resty.kong.log.get\_log\_level](#restykonglogget_log_level)
     * [resty.kong.upstream.set\_next\_upstream](#restykongupstreamset_next_upstream)
+    * [resty.kong.tls.get\_upstream\_request\_ssl\_pointer](#restykongtlsget_upstream_request_ssl_pointer)
+    * [resty.kong.tls.get\_upstream\_certificate](#restykongtlsget_upstream_certificate)
```

Comment:
We already have `get_ssl_pointer` and `get_request_ssl_pointer`. Perhaps it would be better to make the new interface compatible with all these. For example, we can get the peer certificate from the ssl pointer returned by `get_request_ssl_pointer`